CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ This can also be enabled programmatically with `warnings.simplefilter('default',
 ### Added
 - [`set_page_background()`](https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.set_page_background): new method added by @semaeostomea: [link to documentation](https://pyfpdf.github.io/fpdf2/PageFormatAndOrientation.html#per-page-format-orientation-and-background)
 - [`embed_file()`](https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.embed_file) & [`file_attachment_annotation()`](https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.file_attachment_annotation): new methods to add file attachments - [link to documentation](https://pyfpdf.github.io/fpdf2/FileAttachments.html)
+- A new method `set_char_spacing()` allows to increase the spacing between individual characters, thanks to @gmischler; #489
```

Comment:
I must be doing something wrong about forking/branching/rebasing for those entriesto keep appearing in the diffs...